app/controllers/texts_controller.rb

```diff
@@ -1,4 +1,9 @@
 class TextsController < ApplicationController
   def index
+    if params[:genre].in?(genre: params[5..15])
+      @texts = Text.where(genre: params[:genre])
+    else
+      @texts = Text.where(genre: ["Basic", "Git", "Ruby", "Ruby on Rails"])
+    end
```

Comment:
こちらでもないですね。

この部分はこのタスクと直接関係ないので後回しにしましょうか。リンクがないと理由も分かりづらいと思いますので。

```suggestion
@texts = Text.where(genre: ["Basic", "Git", "Ruby", "Ruby on Rails"])
```